onnx/version_converter/convert.cc

```diff
@@ -76,9 +76,9 @@ void DefaultVersionConverter::convert_graph(
                "Please be advised the converted model may not be working properly if target runtime does not support this "
                "experimental op."
             << std::endl;
-        continue;
-      }
-      if (op_name != "Undefined" && op_name != "Captured") {
+      } else if (cur_op->domain() != "" && cur_op->domain() != "ai.onnx") {
```

Comment:
What if a node is Undefined from a domain != from "" or "ai.onnx"?

Reply:
Good question. Did you mean an operator named "Undefined" but from domain "" or "ai.onnx"? If yes, current behavior is do nothing without any error message.